components/LandingPage/LandingPage.js

```diff
@@ -0,0 +1,63 @@
+import { useState } from "react";
+import * as Styled from "./LandingPage.styled";
+import { MdInfoOutline } from "react-icons/md";
+import { AnimatePresence } from "framer-motion";
+
+export default function LandingPage() {
+  const [isInfoBoxVisible, setInfoBoxVisible] = useState(false);
+
+  function handleShowInfo() {
+    setInfoBoxVisible(!isInfoBoxVisible);
+  }
+
+  return (
+    <Styled.InfoContainer>
+      {!isInfoBoxVisible && (
+        <>
+          <Styled.Title>Ruh</Styled.Title>
+          <Styled.Subtitle>
+            Track your moods, <br />
+            get inspired to change them <br />
+            and connect with others
+          </Styled.Subtitle>
+          <Styled.InfoIcon onClick={handleShowInfo}>
+            <MdInfoOutline />
+          </Styled.InfoIcon>
```

Comment:
It's good practice to wrap interactive elements, especially icons that act as buttons, within `<button>` elements. This enhances accessibility and interactivity since buttons inherently provide keyboard focus and semantics for screen readers. 
It's important that the button is given an `aria-label` when it only contains an icon, as the icon alone often doesn't provide enough description. This improves accessibility by providing context to screen reader users.

```suggestion
<button onClick={handleShowInfo} aria-label="More information">
      <MdInfoOutline />
    </button>
```